packages/jellyfish-transaction/src/script/dftx/dftx_masternode.ts

```diff
@@ -7,9 +9,9 @@ import { BufferComposer, ComposableBuffer } from '../../buffer/buffer_composer'
  * CreateMasterNode DeFi Transaction
  */
 export interface CreateMasterNode {
-  type: number // ----------------------------------| 1 byte, if 0x01 p2pkh, otherwise = p2wpkh
-  collateralPubKeyHash: string // ------------------| VarUInt{20 bytes}
-  operatorPubKeyHash?: string // -------------------| VarUInt{20 bytes}, p2pkh only, optional, use collateral pkh as default
+  operatorType: number // --------------------------| 1 byte, 0x01 = p2pkh, 0x04 = p2wpkh
+  operatorAuthAddress: string // -------------------| VarUInt{20 bytes}
+  timeLock?: number // -----------------------------| 4 bytes, only available after EunosPaya
```

Comment:
from c++ side
uint16_t timelock{0};
```suggestion
  timeLock?: number // -----------------------------| 2 bytes, only available after EunosPaya
```